apps/spruce/src/pages/adminSettings/index.tsx

```diff
@@ -0,0 +1,62 @@
+import styled from "@emotion/styled";
+import Icon from "@leafygreen-ui/icon";
+import { Link } from "react-router-dom";
+import { usePageTitle } from "@evg-ui/lib/hooks/usePageTitle";
+import {
+  SideNav,
+  SideNavGroup,
+  SideNavItem,
+  SideNavPageContent,
+  SideNavPageWrapper,
+} from "components/styles";
+import { AdminSettingsProvider } from "./Context";
+import { AdminSettingsTab } from "./Tabs";
+
+const AdminSettings: React.FC = () => {
+  usePageTitle("Admin Settings");
+
+  return (
+    <AdminSettingsProvider>
+      <SideNavPageWrapper>
+        <SideNav aria-label="Admin Settings" widthOverride={250}>
+          {}
+          <ButtonsContainer>{}</ButtonsContainer>
+
+          <SideNavGroup
+            glyph={<Icon glyph="Settings" />}
+            header="Admin Settings"
+          >
+            {}
+            <SideNavItem
+              as={Link}
+              data-cy="navitem-admin-general"
```

Comment:
I think this tab should be called announcements. This can still be the default tab even if we don't name it general.

Reply:
I understand. Let's keep it General 👍

Reply:
I think this tab should be specific to the announcements section. When the user clicks on this tab, the main component that contains all forms besides event-logs and restart-tasks will be loaded and the form will scroll to the announcements section.